gazel.go

```diff
@@ -162,6 +162,7 @@ func (v *Venderor) updatePkg(path, _ string, pkg *build.Package) error {
 
 	var attrs Attrs = make(Attrs)
 	srcs := asExpr(merge(pkg.GoFiles, pkg.SFiles)).(*bzl.ListExpr)
+	cgoSrcs := asExpr(merge(pkg.CgoFiles, pkg.CFiles, pkg.CXXFiles, pkg.HFiles)).(*bzl.ListExpr)
```

Comment:
are we supposed to include header files too in the `srcs` rule?

`cc_library` keeps them separate, but there's no such option for `cgo_library`. :\